content/applications/finance/payment_providers/stripe.rst

```diff
@@ -2,132 +2,132 @@
-
-      .. tip::
-         To use your own API keys, :ref:`activate the Developer mode <developer-mode>` and
-         :ref:`enable Stripe manually <payment_providers/add_new>`. You can then :ref:`Fill in your
-         credentials <stripe/api_keys>`, :ref:`generate a webhook <stripe/webhook>` and enable the
-         payment provider.
+      #. :ref:`Navigate to the payment provider Stripe <payment_providers/supported_providers>` and
+         click :guilabel:`Connect Stripe`.
+      #. Go through the setup process and confirm your email address when Stripe sends you a
+         confirmation email.
+      #. At the end of the process, click :guilabel:`Agree and submit`. If all requested information
+         has been submitted, you are then redirected to Odoo, and your payment provider is enabled.
+      #. :ref:`stripe/local-payment-methods`.
 
    .. group-tab:: Odoo.sh or On-premise
 
-      #. Go to the **eCommerce** or the **Sales** app and click on the *Activate Stripe* or the *Set
-         payments* button on the onboarding banner.
-      #. Fill in the requested information and submit the form.
-      #. Confirm your email address when Stripe sends you a confirmation email.
-      #. At the end of the process, you are redirected to the payment provider **Stripe** on Odoo.
+      #. :ref:`Navigate to the payment provider Stripe <payment_providers/supported_providers>` and
+         click :guilabel:`Connect Stripe`.
+      #. Go through the setup process and confirm your email address when Stripe sends you a
+         confirmation email.
+      #. At the end of the process, click :guilabel:`Agree and submit`; you are then redirected to
+         the payment provider **Stripe** in Odoo.
       #. :ref:`Fill in your credentials <stripe/api_keys>`.
       #. :ref:`Generate a webhook <stripe/webhook>`.
-      #. Enable the payment provider.
-      #. You are all set and can continue to :ref:`stripe/local-payment-methods`.
+      #. Select a :ref:`payment journal <payment_providers/journal>`.
+      #. Set the :guilabel:`State` field to :guilabel:`Enabled`.
+      #. :ref:`stripe/local-payment-methods`.
 
 .. tip::
-   To connect your Stripe account after the onboarding is already completed, go to
-   :menuselection:`Accounting --> Configuration --> Payment Providers --> Stripe` and click on the
-   *Connect Stripe* button.
-
-.. important::
-   If you are testing Stripe (in **test mode**), change the **State** to *Test Mode*. We recommend
-   doing this on a test Odoo database rather than on your main database.
+   - To use an existing Stripe account, :ref:`activate the Developer mode <developer-mode>` and
+     :ref:`enable Stripe manually <payment_providers/add_new>`. You can then :ref:`Fill in your
+     credentials <stripe/api_keys>`, :ref:`generate a webhook <stripe/webhook>`, and enable the
+     payment provider.
+   - You can also try Stripe as a test using the :ref:`payment_providers/test-mode`. To do so,
```

Comment:
```suggestion
   - You can also test Stripe using the :ref:`payment_providers/test-mode`. To do so,
```
"You can also try Stripe as a test using the test mode" is a bit repetitive :) I think this is clearer.